Documentation/operations/upgrade.rst

```diff
@@ -514,6 +514,9 @@ Changed Metrics
   to default deny for an allow policy. Similarly, for an Egress CNP, an empty slice in
   one of the fields ``toEndpoints``, ``toCIDR``, ``toCIDRSet`` and ``toEntities`` will
   not select any identity either.
+* In a similar vein, the behaviour of a CNP with ``toCIDRSet`` or
+  ``fromCIDRSet`` selectors using ``cidrGroupRef`` targeting only non-existent CIDR
+  groups was changed from allow-all to deny-all to align with the above change.
```

Comment:
This bullet depends on the previous one ("In a similar vein", "above change"), what makes the list harder to read an maintain. I think it should be either merged into the previous bullet, or refactored so that it doesn't rely on ordering.

Reply:
good point, thanks!